src/Symfony/Component/VarDumper/Test/VarDumperTestTrait.php

```diff
@@ -19,6 +19,26 @@
  */
 trait VarDumperTestTrait
 {
+    /** @internal */
+    private static $casters = [];
+    /** @internal */
+    private static $flags;
+    /** @internal */
+    private static $varCloner;
```

Comment:
no need for this property - it is cleared after every test case anyway

Reply:
Not if you use `setupBeforeClass`

Reply:
Why would it be needed even in the class case?

Reply:
I that that according to another comment